polaris.shopify.com/src/components/TokensPage/TokensPage.tsx

```diff
@@ -29,24 +28,20 @@ export type NavItem = {
 
 const navItems: NavItem[] = [
   {
-    title: 'Colors',
-    url: `/tokens/colors`,
+    title: 'Color',
+    url: `/tokens/color`,
   },
   {
     title: 'Font',
     url: `/tokens/font`,
   },
   {
-    title: 'Shape',
-    url: `/tokens/shape`,
+    title: 'Border',
+    url: `/tokens/border`,
   },
   {
-    title: 'Spacing',
-    url: `/tokens/spacing`,
-  },
-  {
-    title: 'Depth',
-    url: `/tokens/depth`,
+    title: 'Space',
+    url: `/tokens/space`,
```

Comment:
```suggestion
    title: 'Spacing',
    url: `/tokens/spacing`,
```

Revert handled in Alex's PR